testing/trino-server-dev/etc/catalog/delta.properties

```diff
@@ -0,0 +1,10 @@
+connector.name=delta-lake
+
+# Configuration appropriate for Hive as started by product test environment, e.g.
+#   trino-product-tests-launcher/bin/run-launcher env up --environment singlenode --without-trino
+# On Mac, this additionally requires that you add "<your external IP> hadoop-master" to /etc/hosts
+hive.metastore.uri=thrift://localhost:9083
+hive.hdfs.socks-proxy=localhost:1180
```

Comment:
is this needed, if we're working with MinIO?

Reply:
Sorry, this is a local configuration file which ended into this PR.